### PR TITLE
LPS-143496 Display field error when trying to update node's id with a duplicated id

### PR DESCRIPTION
**Pull request by naraandrade**

### CODE REVIEW

**Comment by naraandrade** [comment]
ci:test:relevant

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -1367,6 +1367,7 @@ anonymous-posting=Anonymous Posting
 anonymous-user-configuration-name=Anonymous User
 anonymous-user-segments-cache-expiration-time-description=Define the time, in seconds, in which the anonymous segment cache is cleared.
 anonymous-user-segments-cache-expiration-time-name=Anonymous User Segments Cache Expiration Time
+another-node-is-already-using-this-id=Another node is already using this ID. Please, define a different ID.
```
Comment (rafaprax) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:1370]: Hey @rbohl, Could you please take a look at this error message and give your feedback on its clarity? Thx Russ

/cc @naraandrade

Comment (rbohl) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:1370]: I think this is slightly clearer and matches more closely with similar keys I see in master:

> A node with that ID already exists. Please enter a unique ID.

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:forward

**Comment by anabuchmann** [comment]
Just started reviewing :)

**Comment by anabuchmann** [comment]
Manually forwarded here: https://github.com/brianchandotcom/liferay-portal/pull/111068

**Pull request closed without merging**